internal/charmstore/search.go

```diff
@@ -210,6 +211,11 @@ func (s *Store) searchDocFromEntity(e *mongodoc.Entity, be *mongodoc.BaseEntity)
 		return nil, errgo.Mask(err)
 	}
 	doc.TotalDownloads = allRevisions.Total
+	if len(doc.Entity.SupportedSeries) == 0 {
```

Comment:
This will only be true for bundles, right?

Wouldn't it be more obvious to make the check specific to "bundle" ?